src/snowflake/cli/api/project/definition_manager.py

```diff
@@ -26,11 +26,10 @@ class DefinitionManager:
     _project_config_paths: List[Path]
 
     def __init__(self, project_arg: Optional[str] = None) -> None:
-        search_path = os.getcwd()
-        if project_arg:
-            search_path = os.path.abspath(project_arg)
-        project_root = DefinitionManager.find_project_root(Path(search_path))
```

Comment:
@sfc-gh-turbaszek can we remove the `find_project_root` as well?

Reply:
NADE commands are using it for checking if project is not within another project